content/en/users/compute/orchestration/im/cli/_index.md

```diff
@@ -106,7 +109,42 @@ configure wn (
 deploy node 1
 ```
 
-Then we can call the `create` operation of the IM client tool:
+IM also supports TOSCA. For example this is an equivalent TOSCA document to
+deploy a single VM::
+
+```yaml
+tosca_definitions_version: tosca_simple_yaml_1_0
+
+imports:
+- indigo_custom_types: https://raw.githubusercontent.com/indigo-dc/tosca-types/master/custom_types.yaml
+
+topology_template:
+
+  node_templates:
+
+    simple_node:
+      type: tosca.nodes.indigo.Compute
+      capabilities:
+        endpoint:
+          properties:
+            network_name: PUBLIC
+        host:
+          properties:
+            num_cpus: 1
+            mem_size: 1 GB
+        os:
+          properties:
+            image: appdb://egi.ubuntu.20.04?vo.access.egi.eu
+
+  outputs:
+    node_ip:
+      value: { get_attribute: [ simple_node, public_address, 0 ] }
+    node_creds:
+      value: { get_attribute: [ simple_node, endpoint, credential, 0 ] }
+```
+
+Then we can call the `create` operation of the IM client tool using the a radl
+or a TOSCA yaml file:
```

Comment:
is the same `create` operation for yaml or radl? does the user need to specify anything?

Reply:
It is the same create operation. The user does no have to specify anything. 
The client assumes the content of the file from the file extesion (.radl or .yml/yaml).